src/main/java/dev/emi/trinkets/mixin/LivingEntityMixin.java

```diff
@@ -128,10 +130,12 @@ private void dropInventory(CallbackInfo info) {
 		}));
 	}
 
+	@Unique
 	private void dropFromEntity(ItemStack stack) {
 		ItemEntity entity = dropStack(stack);
 		// Mimic player drop behavior for only players
-		if (entity != null && ((Entity) this) instanceof PlayerEntity) {
+        //noinspection ConstantValue
```

Comment:
I prefer to not use noinspection in my codebase

Reply:
Ok